src/m365/spo/commands/listitem/listitem-get.spec.ts

```diff
@@ -177,7 +177,7 @@ describe(commands.LISTITEM_GET, () => {
       webUrl: 'https://contoso.sharepoint.com/sites/project-x', 
       id: expectedId,
       output: "json",
-      fields: "ID,Modified"
+      properties: "ID,Modified"
```

Comment:
We should update all tests so that they no longer refer to fields but to properties to keep the test suite consistent with the new code